Mathlib/LinearAlgebra/Dimension.lean

```diff
@@ -935,6 +935,14 @@ theorem rank_eq_card_chooseBasisIndex : Module.rank K V = #(ChooseBasisIndex K V
   (chooseBasis K V).mk_eq_rank''.symm
 #align module.free.rank_eq_card_choose_basis_index Module.Free.rank_eq_card_chooseBasisIndex
 
+lemma rank_eq_mk_of_infinite_lt [Infinite K] (h_lt : lift.{v, u} #K < lift.{max u v, v} #V) :
```

Comment:
The minor-est nit -- should `K` and `V` be implicit here? Won't be useful in your `rank_rat_*`, but according to guidelines, they're both present in `h_lt`.

Reply:
And I think the condition is usually written simply as `lift.{v} #K < lift.{u} #V` (i.e. `lift.{v u} #K < lift.{u v} #V`. This is the case, for example, in [this theorem you use](https://github.com/leanprover-community/mathlib4/blob/96f0a9003a1032e51a21b807aa9fe772112b5816/Mathlib/SetTheory/Cardinal/Ordinal.lean#L1049-L1056).